docs/cpp-runnable.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Generating Executables
+id: cpp-runnable
+---
+
+The `fuse` CLI provides the `run` subcommand for generating executables that
+can be used to test Fuse code without going throught a synthesis workflow.
```

Comment:
More saliently, I'd say it's useful for testing Fuse code without writing your own host-side wrapper code (i.e., `main` function).